projects/vdk-plugins/vdk-properties-fs/src/vdk/plugin/properties_fs/fs_properties_client.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+import logging
+import os
+from configparser import ConfigParser
+from typing import Dict
+
+from vdk.api.plugin.plugin_input import IPropertiesServiceClient
+
+log = logging.getLogger(__name__)
+
+
+class FileSystemPropertiesServiceClient(IPropertiesServiceClient):
+    """
+    Implementation of IProperties that are kept on local FS only. Stateless.
+    A single properties file store, supporting multiple teams and data jobs properties,
+    via placing the key-values in a dedicated team and job name section to deduplicate.
+    """
+
+    def __init__(self, directory: str, filename: str):
+        self._file_path = os.path.join(directory, filename)
+
+    def read_properties(self, job_name: str, team_name: str) -> Dict:
+        config_parser = ConfigParser()
+        config_parser.read(self._file_path)
```

Comment:
Wouldn't this fail if you try to read properties before writing any? Not sure if ConfigParser.read fails if the provided file path doesn't exist, maybe my comment is wrong.

Reply:
Yeah it shouldn't fail.
from the docs 
```
        Files that cannot be opened are silently ignored; this is
        designed so that you can specify an iterable of potential
        configuration file locations (e.g. current directory, user's
        home directory, systemwide directory), and all existing
        configuration files in the iterable will be read.
```